### PR TITLE
feat: create layer for documentdb

### PR DESCRIPTION
**Pull request by dschro-1993**
Fixes #

### CODE REVIEW

File: graphql/lib/documentdb.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    await client
+      .connect()
+      .catch((reason) => {
+        console.error(reason);
+        throw reason;
+      });
+  }
+}
+
+// const validator = {
```
Comment (gdenn) [inline graphql/lib/documentdb.ts:24]: left-over comment

File: graphql/lib/documentdb.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  dotenv.config();
+  await connectToDB();
+  const db = client.db(process.env.DB_NAME!);
+  if (validationOptions) {
+    applyValidation(db, name, validationOptions);
+  }
+  return db.collection<T>(name);
+}
+
+// (async () => {
```
Comment (gdenn) [inline graphql/lib/documentdb.ts:80]: left-over comment

**Review by gdenn — APPROVED** [review]
Looks lgt 💯

**Comment by dschro-1993** [comment]
Paging still missing + test-setup, will add this tomorrow.

File: graphql/lib/documentdb.ts (hunk 1 above)
Comment (dschro-1993) [reply graphql/lib/documentdb.ts:24]: just serves as an example.

**Comment by gdenn** [comment]
@dschro-1993 let us merge this rather sooner than later. This is useful for continuing my work.

**Pull request merged**